### DISCUSSION TITLE
Any examples of design systems in the wild using vanilla extract

### DISCUSSION THREAD

**Discussion by coreybruyere** [vanilla-extract-css/vanilla-extract#437] [General]
Looking for some open sourced design system inspiration. Besides Seek, anyone else have some example repos to share?

**Comment by timoclsn** [comment]
Not really building a design system for a specific purpose but I'm currently exploring (in the open) patterns on how to build some foundational components with vanilla-extract: 

https://github.com/timoclsn/mauli-vanilla-extract

Don't mind the visuals for now - just exploring the technical side:
- Theming
- Design tokens
- Atomic Styles
- Box component
- Responsive props
- Capsized text
- …

Would also love some more examples!

_(Of course taking a lot of inspiration from Braid 🙏)_

**Comment by timoclsn** [comment]
I think Shopify is also doing a bit of exploration right here:

https://github.com/Shopify/foundational-design-system-proto

cc @alex-page

**Comment by tmm** [comment]
Been working on one at mirror.xyz

https://github.com/mirror-xyz/degen

**Reply by gaboesquivel** [reply]
nice

**Comment by toha993** [comment]
https://github.com/optimizely-axiom/optiaxiom
